airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/uploader/AbstractBigQueryUploader.java

```diff
@@ -120,8 +117,6 @@ protected void uploadData(final Consumer<AirbyteMessage> outputRecordCollector,
     } catch (final Exception e) {
       LOGGER.error("Upload data is failed!");
       throw e;
-    } finally {
-      dropTmpTable();
```

Comment:
why weren't we hitting this clause previously 🤔 

(it's better to just not create these tables to begin with, I'm just curious)

Reply:
I think because we stopped using this method - the table gets created in `BigQueryDestination` which we still use in v2, but there is probably a larger clean up effort to be done here